handsontable/src/dataMap/metaManager/metaSchema.js

```diff
@@ -3371,7 +3369,7 @@ export default () => {
      * Using the [`rowHeights`](#rowHeights) option forcibly disables the [`AutoRowSize`](@/api/autoRowSize.md) plugin.
      *
      * Read more:
-     * - [`AutoRowSize`](@/api/autoRowSize.md)
+     * - [`AutoRowSize`&#8594;](@/api/autoRowSize.md)
```

Comment:
Is this change intentional?

Reply:
@wszymanski - yup, a minor thing I spotted and wanted to fix straight away